docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

```diff
@@ -219,20 +221,35 @@ This workflow is going to serve as the base installation mechanism for our self-
 
 As many other OCP install workflows it will place the kubeconfig file and other information about the cluster in the $SHARED_DIR available to all containers.
 
-##### `cucushift-installer-rehearse-aws-ipi-spot`<!-- omit from toc -->
+We have the option to set an env variable in the config to use AWS spot instances to save costs in development.
 
-step-registry doc: [cucushift-installer-rehearse-aws-ipi-spot](https://steps.ci.openshift.org/workflow/cucushift-installer-rehearse-aws-ipi-spot)
-
-This workflow will serve as our development workflow when installing a self-managed cluster on AWS. This workflow uses AWS spot instances which can significantly decrease our cloud costs.
+```yaml
+    env:
+      SPOT_INSTANCES: "true"
+```
 
 > **IMPORTANT:**
 > 
-> This workflow is not to be used in production.
+> Spot instances are not to be used in production.
 
 ##### Important Workflow.Next<!-- omit from toc -->
 
 As this level of testing expands to different flavors of OCP installations we will cover future important workflows here for reference.
 
+### Multiple private repo environment (Images)
+
+You may run into a situation where PQE needs many different test environments to execute all of their tests. This becomes difficult when the images (read Dockerfiles) are stored in private github repos. We are not able to build all of these images from a single config file through the private repo access being offered by OpenShift CI. This is because the only private repo that we will have access to is the "src" repo (the repo that is associated with the directory naming convention). We also cannot just store all of the Dockerfiles into this one "src" repo and hope to git clone the files from the other private repos because this would require a personal access token (PAT) to clone these files. This PAT can only be implemented in the commands.sh script run against the container rather than at build time of the image itself where we would want to use it.
```

Comment:
```suggestion
You may run into a situation where PQE needs many different test environments to execute all of their tests. This becomes difficult when the images (read Dockerfiles) are stored in private github repos. We are not able to build all of these images from a single config file through the private repo access being offered by OpenShift CI. This is because the only private repo that we will have access to is the "src" repo (the repo that is associated with the directory naming convention). We also cannot just store all of the Dockerfiles into this one "src" repo and hope to git clone the files from the other private repos because a personal access token (PAT) would be required. If we tried using the [secrets functionality built into OpenShift CI](../Secrets/Secrets_Guide.md), the PAT would only be accessable during the execution of a ref, not during the image's build-process.
```